src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -1138,7 +1138,7 @@ public File get() {
                     250, TimeUnit.MILLISECONDS,
                     1024,
                     true,
-                    32 * 1024,
+                    FileUtils.ONE_MB * 10,
```

Comment:
isn't this just a tad excessive (50MB of event processing logs)?

if previously the size of logs with your load would keep 15 minutes then this would hold 305 times that (or 76 hours worth or a tad over 3 days).  I would expect that 8 hours would be enough which would be (assuming all my calculation are correct) :

```suggestion
                    FileUtils.ONE_MB,
```